src/components/Icon/Icon.jsx

```diff
@@ -118,6 +135,7 @@ const Icon = createClass({
 					},
 					className
 				)}
+				onClick={this.handleSelect}
```

Comment:
Is `onClick` the correct term to use here? If the goal is to create an `onClick` handler but to also prevent overriding any other handlers downstream, we use the term `onSelect` for the handler but `onClick` for the actual action, correct?

Reply:
`onClick` must be used here since it's on an `svg` html element which accepts `onClick` (see https://reactjs.org/docs/dom-elements.html#all-supported-html-attributes)

Reply:
You're correct to be using `onClick` here. For clarity I'd probably make this `onClick={this.handleClick}` and rename your callback function to `handleClick` (which will call the user's `onSelect` prop)